### ISSUE TITLE
FEATURE REQUEST (bug?) - speed up uninstall

### ISSUE THREAD

**Issue by bcurran3** [chocolatey/ChocolateyGUI#687]
ChocolateyGui 0.16.0 (February 15, 2018)

This "bug" might be intentional, but it seems weird to me.

I used ChocolateyGui to look through my installed packages for a little clean up of things I don't use. I noticed I had 7zip.commandline which is deprecated so I right clicked to uninstall it. In the output, other packages are mentioned and I have no understanding why. Three are packages of mine that are 1 rejected, 1 in moderation, and 1 in -pre status. The other is not a package of mine and may or may not be outdated. 

Basically when I click to uninstall a package via ChocolateyGUI, I expect it's going to run the console and do "choco uninstall packagename" and not search for or do whatever other processing it just did.

![image](https://user-images.githubusercontent.com/14026600/59970591-53904780-951f-11e9-8e63-d71dd2ff3ce4.png)

EDIT: lyricsfinder had an update available.

I have NOT noticed this behavior with other packages. It may be unique due to the package trying to be uninstalled was deprecated. I'm baffled by the behavior.

**Comment by gep13** [comment]
@bcurran3 what you are seeing here is a "symptom" of the fact that Chocolatey GUI v0.16.0  runs the `choco outdated` command several times, in order to keep the information about which packages are out of date up to date.

This is actually a duplicate of an issue that was raised here: https://github.com/chocolatey/ChocolateyGUI/issues/525 and which has been solved in the soon to be released 0.17.0 version of Chocolatey GUI.  Could I ask that you try out the latest pre-release version, and let me know if you have any additional comments.

For now, I am going to go ahead and close this, as I do believe that it is a duplicate.
